docs/settings.rst

```diff
@@ -1044,6 +1044,18 @@ more information.
 
    The default language to use.
 
+.. data:: ARTICLE_TRANSLATE_PER = {'slug'}
+
+   The metadata attribute(s) used to identify which articles are translations
+   of one another. May be a string or a collection of strings. Set to ``None``
+   or ``False`` to disable the identification of translations.
+
+.. data:: PAGE_TRANSLATE_PER = {'slug'}
+
+   The metadata attribute(s) used to identify which pages are translations
+   of one another. May be a string or a collection of strings. Set to ``None``
+   or ``False`` to disable the identification of translations.
+
```

Comment:
Even though I get the intention, `set`s aren't the most known collections in python. I'm a little hesitant to expose them to the user like that. A `list` or a `tuple` would be more intuitive I think. Or defaults can be simply
```
..._PER = 'slug'
```